doc.go

```diff
@@ -0,0 +1,58 @@
+/*
+Package fluent is an ORM build around the database/sql package.
+
+Overview
+	- SQL builder
+	- Supported database(s): Postgres
+
+Quickstart
+
+Before you can use the Fluent ORM you have to connect to a database, for example:
+
+	import (
+		"database/sql"
+		_ "github.com/lib/pq"
+		bitbucket.org/sebas_dk/fluent
+
+	)
+	func main() {
+		db, err := sql.Open("postgres", "postgres://pqgotest:password@localhost/pqgotest?sslmode=verify-full")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		fluent := fluent.New(db)
+		…
+	}
+
+Query Builder
+
+The Fluent ORM uses the tags in the struct to find and set values.
+	type Test struct {
+	    ID int       `sql:"id"`
+	    Name string  `sql:"name"`
+	    Total string `sql:"total"`
+	}
+
+Create Record
+  test := Test{Name: "user_1", Total: 12.00}
+  fluent.Table("test").Insert(test)
+
+Update Record
+  test := Test{Name: "user_2"}
+  fluent.Table("test").Where("id","=", 1).Update(test)
+
+Fetch Record
+  test := Test{}
+  fluent.Table("test").Where("id","=", 1).Get([]string("id","name","total").One(&test)
```

Comment:
It would be nice to return the result rather than passing a reference to bind the result to
`record := fluent.Table("test").Where("id","=", 1).Get([]string("id","name","total").One()`

also the `Get` method params could be Variadic Parameters so we can query like
`record := fluent.Table("test").Where("id","=", 1).Get("id","name","total").One()`

Personally I would prefer `First()` vs `One()`